src/restic/node.go

```diff
@@ -34,6 +41,7 @@ type Node struct {
 	Size       uint64      `json:"size,omitempty"`
 	Links      uint64      `json:"links,omitempty"`
 	LinkTarget string      `json:"linktarget,omitempty"`
+	Xattrs     []Xattr     `json:"xattrstore,omitempty"`
```

Comment:
I think the name `xattrstore` for the JSON structure is not a good fit. What about `extended_attributes`? And use `ExtendedAttributes` for the structure variable name, what do you think?